src/config.rs

```diff
@@ -286,7 +291,7 @@ impl Config {
                         Value::String(s) => {
                             let extension = Path::new(&s).extension();
                             if extension.is_some() && extension.unwrap() == "jar" {
-                                config.apktool_file = s.clone();
+                                config.apktool_file = PathBuf::from(s.clone());
```

Comment:
Turns out, that is also not possible, thus I'm sticking with original implementation if that's ok?